chain/network/src/network_protocol.rs

```diff
@@ -424,3 +424,8 @@ impl PeerMessage {
         }
     }
 }
+
+const _: () =
+    assert!(std::mem::size_of::<HandshakeFailureReason>() <= 64, "HandshakeFailureReason");
+const _: () = assert!(std::mem::size_of::<PeerMessage>() <= 1144, "PeerMessage");
+const _: () = assert!(std::mem::size_of::<RoutingSyncV2>() <= 80, "RoutingSyncV2");
```

Comment:
I'd probably put each of those after the corresponding struct. That way, when you look at the type's definiion, you realize that this is the place where we care about size.